### PR TITLE
Switch to Github action

### PR DESCRIPTION
**Pull request by seuros**
Test rails 7.0.0
Normalize testing suite

### CODE REVIEW

File: test/dummy/app/concepts/song/contract/form.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module Song::Contract
```
Comment (apotonick) [inline test/dummy/app/concepts/song/contract/form.rb:1]: This should be named either `Song::Contract::Create` or `Song::Form::Create`.

File: trailblazer-rails.gemspec [hunk 2]
```diff
@@ -9,16 +9,16 @@ Gem::Specification.new do |spec|
   spec.email         = ["apotonick@gmail.com"]
 
   spec.summary       = "Convenient Rails support for Trailblazer."
-  spec.homepage      = "http://trailblazer.to/gems/trailblazer/2.0/rails.html"
+  spec.homepage      = "https://trailblazer.to/gems/trailblazer/2.0/rails.html"
```
Comment (apotonick) [inline trailblazer-rails.gemspec:12]: This should be https://trailblazer.to/2.1/docs/trailblazer.html#trailblazer-rails

**Review by apotonick — CHANGES_REQUESTED** [review]

File: test/integration/artists_controller_test.rb [hunk 3]
```diff
@@ -3,15 +3,15 @@
 class ArtistsControllerTest < Minitest::Capybara::Spec
   it "cell: Artist::Cell::Dashboard" do
     visit "/artists/dashboard"
-    page.must_have_css "h1", visible: "Dashboard"
-    page.must_have_css "h2", visible: "LAYOUT"
+    page.must_have_css "h1"
```
Comment (apotonick) [inline test/integration/artists_controller_test.rb:6]: Why are we changing the test here?

File: test/integration/songs_controller_test.rb [hunk 4]
```diff
@@ -7,22 +7,22 @@ class SongsControllerTest < Minitest::Capybara::Spec
   end
 
   it "show" do
-    song = Song::Create.(params: {title: "Skin Trade"})[:model]
+    song = Song::Operation::Create.call(params: {title: "Skin Trade"})[:model]
 
     visit "/songs/#{song.id}"
-    page.must_have_css "h1", visible: "Skin Trade"
+    page.must_have_css "h1"
   end
```
Comment (apotonick) [inline test/integration/songs_controller_test.rb:14]: Why are we removing this? Isn't that dangerous and almost "wrong"?

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    branches:
+      - master
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.7, '3.0']
+        gemfile:
+          - rails_6.0
```
Comment (apotonick) [inline .github/workflows/ci.yml:16]: + Rails 5.2 please!!!!

**Review by apotonick — CHANGES_REQUESTED** [review]

File: test/integration/artists_controller_test.rb (hunk 3 above)
Comment (seuros) [reply test/integration/artists_controller_test.rb]: new syntax.

**Review by apotonick — APPROVED** [review]

**Pull request merged**